src/graphql/schemas/typeDefs/hypercertTypeDefs.ts

```diff
@@ -43,7 +54,7 @@ class Hypercert extends HypercertBaseType {
     nullable: true,
     description: "The metadata for the hypercert as referenced by the uri",
   })
-  metadata?: Metadata;
+  declare metadata?: Metadata;
```

Comment:
Because typescript/intellij told me so, as TS was throwing a warning that we were accidentally overwriting a property. This makes that intentional. The type needs to be there because otherwise graphQL cant deduce the types.

Reply:
That's because you are overwriting a property, please review the base types

Reply:
Created https://github.com/hypercerts-org/hypercerts-api/issues/128.